packages/docs/src/helpers/SortObjectByPropNamesHelper.ts

```diff
@@ -0,0 +1,9 @@
+export const sortObjectByPropNames = (obj: any) => {
+  const arrFromObj = Object.entries(obj);
+
+  const sortedArr = arrFromObj.sort((element1: any, element2: any) => {
```

Comment:
Once again, no `any`s. It's gonna live like this for quite some time definitely and no one's gonna fix it if you don't do it now.